lib/Dialect/AIE/IR/AIETargetModel.cpp

```diff
@@ -289,6 +289,11 @@ bool AIE1TargetModel::isLegalTileConnection(int col, int row,
   return false;
 }
 
+bool AIE1TargetModel::hasProperty(ModelProperty prop) const {
+  uint64_t properties = 0;
+  return (properties & prop) == prop;
+}
+
```

Comment:
This doesn't add anything and can be covered by the base class `AIETargetModel` ?

Reply:
Agreed. Now removed in PR.